app/src/lib/stores/app-store.ts

```diff
@@ -3059,6 +3059,11 @@ export class AppStore extends TypedBaseStore<IAppState> {
     const { tip } = branchesState
     const hasChanges = changesState.workingDirectory.files.length > 0
 
+    // No point in checking out the currently checked out branch.
+    if (tip.kind === TipState.Valid && tip.branch.name === branch.name) {
```

Comment:
@**
```suggestion
    if (tip.kind === TipState.Valid && tip.branch.name === branch.name) {
```suggestion
    if (tip.kind === TipState.Valid && tip.branch.name === branch.name) {![default.jpg](https://user-images.githubusercontent.com/46066539/102594304-dd947e80-40ca-11eb-9fe7-1f85389b0b6c.jpg)
```

```
**

Reply:
The Green Post Office is open